admin-cli/src/index.ts

```diff
@@ -30,36 +30,92 @@ function connect(config: Config, party: string): Ledger {
   return new Ledger(token, config.ledgerUrl);
 }
 
+namespace v3 {
+  export async function init (config : Config): Promise<void> {
+    const companyLedger = connect(config, config.company);
+    for (const employee in config.employees) {
+      const employeeInfo = config.employees[employee];
+      const employeeRole: davl3.EmployeeRole = {
+        company: config.company,
+        employee: employee,
+        boss: employeeInfo.boss,
+      };
+      const employeeProposal: davl3.EmployeeProposal = {
+        employeeRole,
+        vacationDays: employeeInfo.vacationDays.toString(),
+      };
+      const employeeProposalContract =
+        await companyLedger.create(davl3.EmployeeProposal, employeeProposal);
+      console.log(`Created EmployeeProposal for ${employee}.`);
+      if (employeeInfo.acceptProposal) {
+        const employeeLedger = connect(config, employee);
+        await employeeLedger.exercise(
+          davl3.EmployeeProposal.EmployeeProposal_Accept,
+          employeeProposalContract.contractId,
+          {},
+        );
+        console.log(`Accepted EmployeeProposal for ${employee}.`);
+      }
+    }
+  }
+}
+
+namespace v3v4 {
+  export async function upgrade (config: Config): Promise<void> {
+    const companyLedger = connect(config, config.company);
+    const employees = await companyLedger.query(davl3.EmployeeRole, {});
+    for (const employeeRole of employees) {
+      const employee = employeeRole.key;
+      const employeeProposal: davlUpgradev3v4.UpgradeProposal = {
+        employee:employee,
```

Comment:
```suggestion
        employee: employee,
```
These ain't named arguments in OCaml. 😉

Reply:
✔️